cardano-cli/src/Cardano/CLI/Shelley/Parsers.hs

```diff
@@ -1010,6 +1012,17 @@ pQueryCmd =
       <*> pNetworkId
       <*> many pStakePoolVerificationKeyHash
 
+    pQueryTxMempool :: Parser QueryCmd
+    pQueryTxMempool = QueryTxMempool
+      <$> pConsensusModeParams
+      <*> pNetworkId
+      <*> pTxMempoolQuery
+      where
+        pTxMempoolQuery :: Parser TxMempoolQuery
+        pTxMempoolQuery = subparser $
+             command "info" (info (pure TxMempoolQueryInfo) (progDesc "Mempool info"))
```

Comment:
So generally we implement these as individual parsers (combined with `(<|>)` ) that use `help`  from opt-parse. You can copy and paste the haddocks on  `LocalTxMonitoringQuery` as the help text.

Reply:
I'm not completely sure what you wanted here. I think the new version matches the style that's already in the codebase for subcommands. If you want them as flags, I can change that, but since exactly one must always be passed, I think a subcommand is better here.